src/io/probst/idea/clangformat/ClangFormatAction.java

```diff
@@ -165,10 +158,7 @@ public String call() throws Exception {
   private ProcessBuilder getCommand(String filePath, int cursor, int selectionStart,
       int selectionLength) throws ExecutionException, InterruptedException {
     Settings settings = Settings.get();
-
-    ProcessBuilder command = new ProcessBuilder().command(settings.clangFormatBinary, "-style=file",
-        "-output-replacements-xml", "-assume-filename=" + filePath, "-cursor=" + cursor,
-        "-offset=" + selectionStart, "-length=" + selectionLength);
+    ProcessBuilder command = new ProcessBuilder().command(settings.clangFormatBinary,filePath, "-style=file", "-cursor=" + cursor);
```

Comment:
This is a pretty substantial change. It means clang-format only formats the file on disk - all the offset handling above or ranges will no longer work, and you also can no longer format modified buffers without writing them first. I don't think this is a good change, TBH.

Didn't you say you found an issue where it just calculated an offset of a character incorrect wrt the encoding?